lerobot/common/datasets/push_dataset_to_hub/_download_raw.py

```diff
@@ -148,21 +123,30 @@ def download_umi(raw_dir: Path):
 if __name__ == "__main__":
     data_dir = Path("data")
     dataset_ids = [
+        "pusht_image",
```

Comment:
I know it might be a pain to change at this point, but have you considered putting the "image" in the versioning system somehow? So you'd have like `v1.4` and `v1.4i` or something like that.

Reply:
My first impression was that managing multiple version identifiers might complicate the versioning system and lead to inconsistencies, I think we should stick to one versioning

Reply:
@alexander-soare We use the version of the dataset as a hack for backward compatibility. It's actually the version of the code base used to produced the dataset.

@alexander-soare In term of naming, I agree it's a bit confusing for pusht, but I think it's ok since we have the same naming for everythng:
- pusht <--- video version that everyone should use
- pusht_image <--- original version uint8 uncompressed frame by frame
- pusht_no_camera <--- original version with full state provided, no images from cameras

Reply:
Sounds good. Thanks for addressing it :)